SPECS-EXTENDED/libvorbis/libvorbis.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
- "Signatures": {
-  "libvorbis-1.3.6.tar.xz": "af00bb5a784e7c9e69f56823de4637c350643deedaf333d0fa86ecdba6fcb415"
- }
+  "Signatures": {
```

Comment:
Although we did not touch any go code in this PR, it seems maybe a prior PR might have broken the tests.
Lets rebase with `main`.